### PR TITLE
Fix zoom param, cms dependencies

### PR DESCRIPTION
**Pull request by kookster**
- [x] Fix zoom param to propagate to checking for including properties
- [x] Have zoom param returned as an empty list when it has "" or nil value
- [x] Use ActiveRecord includes in index as well as show actions
- [x] Add in base paged collection representer, and make sure it is not hard coded
- [x] Get rid of piece and story references
- [x] Add in useful `id_from_url` method

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,7 +24,51 @@ Or install it yourself as:
 
 ## Usage
 
-TODO: Write usage instructions here
```
Comment (cavis) [inline README.md]: Yeaaaaah docs!

File: lib/hal_api/controller/resources.rb [hunk 2]
```diff
@@ -82,15 +82,12 @@ def find_base
...
 
   def decorate_query(res)
-    filtered(paged(sorted(scoped(res))))
+    filtered(paged(sorted(scoped(included(res)))))
   end
 
   def filtered(arel)
     keys = self.class.resources_params || []
     where_hash = params.slice(*keys)
-    if where_hash.key?('story_id')
```
Comment (cavis) [inline lib/hal_api/controller/resources.rb]: Yeah, this seems like a strange place for this.  (In the gem).

File: lib/hal_api/paged_collection_representer.rb [hunk 3]
```diff
@@ -0,0 +1,63 @@
+require 'hal_api/representer'
+
+class HalApi::PagedCollectionRepresenter < HalApi::Representer
```
Comment (cavis) [inline lib/hal_api/paged_collection_representer.rb]: Moving this out of CMS, it seems.  :+one:!

**Comment by kookster** [comment]
thanks @cavis! Bumped and released to rubygems as `0.3.0`

**Pull request merged**